index.html

```diff
@@ -0,0 +1,874 @@
+      <div class="d-lg-flex">
+        <h2 class="f-category">Newsweek</h2>
+        <p class="text-muted small text-uppercase align-self-end">© 2020 Newsweek</p>
+      </div>
+      <div class="d-flex align-self-end">
+        <a href="#" class="icons"><i class="fa fa-facebook-official"></i></a>
+        <a href="#" class="icons"><i class="fa fa-twitter-square"></i></a>
+        <a href="#" class="icons"><i class="fa fa-instagram"></i></a>
+        <a href="#" class="icons"><i class="fa fa-tumblr"></i></a>
+        <a href="#" class="icons"><i class="fa fa-linkedin"></i></a>
+      </div>
+    </section>
+    <hr>
+    <!-- Edtions list -->
+    <section class="">
+      <div class="d-flex">
+      <h3 class="h5 text-secondary font-weight-bold mr-4">Editions:</h3>
+      <ul class="list-inline text-muted">
+        <li class="list-inline-item"><a href="#">U.S. Edition</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">Romania</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">Polska</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+      </ul>
+      </div>
+      <ul class="list-inline">
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+        <li class="list-inline-item"><a href="#">About us</a><span class="text-secondary d-sm-none d-md-inline"> |</span></li>
+      </ul>
+    </section>
+  </footer>
```

Comment:
- [ ] The footer has a 100% width and it should have the same width as the rest of the page (not including the navbar). Please make the adjustment.